test/AdTest.java

```diff
@@ -0,0 +1,29 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class AdTest {
+    private AdGroup classUnderTest;
+
+    @Before
+    public void setup() {
+        classUnderTest = new AdGroup();
+    }
+
+    @Test
+    public void shoulReturnTheAdTypeThatWasSet() {
+        classUnderTest.setAdType("testAdType");
+        String tstAdType = classUnderTest.getAdType();
+        assertEquals("AdType if not what expected", "testAdType", tstAdType);
```

Comment:
It is preferable to use assertThat and not assertEqual. It is more informative and you don't need to add a message for each test. The first parameter is the expected value and the second one is the parameter you check.